features/steps/suppliers_steps.py

```diff
@@ -14,15 +14,15 @@
 @given('the following suppliers')
 def step_impl(context):
     """ Delete all suppliers and load new ones"""
-    headers = {"Content-Type": "application/json"}
-    context.resp = requests.get(context.base_url + '/suppliers', headers=headers)
```

Comment:
Why we remove the headers parameter?

Reply:
because of Restx , its no longer needed .

Reply:
hm but why there is still header in line 21?

Reply:
I think you are missing the fact that instead of getting header for line line 17 and 18 , its coming from api key as header .

Reply:
you update the header by adding the key on line 17, then we should pass in the header parameter on line 18 get request